gridt/src/app/settings/settings.page.ts

```diff
@@ -50,17 +57,9 @@ export class SettingsPage implements OnInit  {
   @ViewChild('emailfield', {static: true}) emailInput;
   @ViewChild('passwordfield', {static: true}) passwordInput;
 
-  // When pulling down on page the page is refreshed
-  public mobileRefresh(event) {
-    this.refreshPage();
-    event.target.complete();
-  }
-
-  public refreshPage() {
-    this.SetService.getUserSettings();
-    if (this.isDisabled$) {
-      this.serverWarning()
-    }
+  public refreshPage(event?) {
+    this.SetService.updateUserSettings();
```

Comment:
Fun feature that could be built here: if you know you are waiting for a server call, you can connect a loading spinner to the observable of the request. That helps the user understand that he is waiting "for the internet". Not necessary and maybe hard to implement right now, but it could be nice.

Reply:
Actually the fact that it is hard to implement tells me something is off. Probably here you actually do not wish to call `updateUserSettings`, as it also pulls from the localStorage, what you really want is to just make the server call here.